src/message_publisher.cpp

```diff
@@ -366,7 +363,14 @@ void MessagePublisher::publishIMUData(const SbgBinaryLogData &ref_sbg_log)
   }
   if (m_velocity_pub_)
   {
-    m_velocity_pub_.publish(m_message_wrapper_.createRosTwistStampedMessage(m_sbg_imu_message_, sbg_message_imu_old));
+    if (m_sbgEkfQuat_pub_)
```

Comment:
It has nothing to do with IMU data so it should be re-organized.
It's the same in method defineRosStandardPublishers
Also we should make sure members / variables conventions are consistent such as m_SbgAirData_pub_